autopilot/betweenness_centrality.go

```diff
@@ -71,15 +71,22 @@ type BetweennessCentrality struct {
 }
 
 // NewBetweennessCentralityMetric creates a new BetweennessCentrality instance.
-// Users can specify the number of workers to use for calculating centrality.
-func NewBetweennessCentralityMetric(workers int) (*BetweennessCentrality, error) {
+func NewBetweennessCentralityMetric() *BetweennessCentrality {
+	return &BetweennessCentrality{
+		workers: 1,
```

Comment:
why not take the num workers as an argument anymore?

Reply:
We're instantiating the heuristics in during static init, where we cannot really deal with the errors.